src/MorphTransition.js

```diff
@@ -36,8 +36,9 @@ class MorphTransition extends React.Component {
     render() {
         var width = this.props.width;
         var height = this.props.height;
+        var className = this.props.className;
         return (
-            <svg viewBox={this.viewBox} preserveAspectRatio="xMidYMid meet" fit width={width} height={height}>
+            <svg viewBox={this.viewBox} preserveAspectRatio="xMidYMid meet" fit width={width} height={height} className={className}>
```

Comment:
Maybe we should just use object destructuring here and let consumers pass any prop they like, ex: `style`, `onClick`.

```
<svg {...this.props} viewBox={this.viewBox} preserveAspectRatio="xMidYMid meet" fit>
```